tests/Doctrine/Tests/DBAL/Functional/MasterSlaveConnectionTest.php

```diff
@@ -1,243 +0,0 @@
-<?php
-
-namespace Doctrine\Tests\DBAL\Functional;
-
-use Doctrine\DBAL\Connections\MasterSlaveConnection;
-use Doctrine\DBAL\Driver\Statement;
-use Doctrine\DBAL\DriverManager;
-use Doctrine\DBAL\Schema\Table;
-use Doctrine\Tests\DbalFunctionalTestCase;
-use Throwable;
-
-use function array_change_key_case;
-use function sprintf;
-use function strlen;
-use function strtolower;
-use function substr;
-
-use const CASE_LOWER;
-
-/**
- * @group DBAL-20
- */
-class MasterSlaveConnectionTest extends DbalFunctionalTestCase
```

Comment:
these tests should be kept (as legacy tests), to ensure that the deprecated class still works fine.